Notes/ViewModels/ContentViewModel.swift

```diff
@@ -12,20 +12,48 @@ class ContentViewModel: ObservableObject {
     @Published var usernameInput: String = ""
     @Published var passwordInput: String = ""
     @Published var errorMessage = ""
+    @Published var usernameInvalid = false
+    @Published var passwordInvalid = false
     @Published var isLinkActive = false
+<<<<<<< HEAD:Notes/ContentViewModel.swift
+        
+	enum ScreenEvent {
+		case signIn
+	}
+	
+	func onScreenEvent(_ event: ScreenEvent) {
+		switch event {
+			case .signIn:
+				usernameEqualToInput()
+		}
+	}
+	
+	
+=======
     
     let footnoteFont = Font.system(.footnote, design: .monospaced)
     //let presistenceController = PersistenceController()
     
+>>>>>>> parent of e5af394 (Revert "Fetch request"):Notes/ViewModels/ContentViewModel.swift
```

Comment:
there is something wrong with this. Looks like there were git conflict not resolved